source/publishing_data/06_configuring_metadata/standard_metadata.rst

```diff
@@ -80,13 +82,24 @@ Refer to the table below to know more about the available metadata and how to co
     * Language of the data in the dataset.
     * 1. Click on the drop down menu.
       2. Choose a language amongst the Domain languages or other languages available.
-  * * Geographic area
-    * Geographic area of the data contained in the dataset. This metadata is directly linked to :ref:`the mini map displayed in the front office of the portal <using-the-minimap>` : if the geographic area metadata is set for a dataset, an end-user will be able to find it through a geographic search of the catalog.
+  * * Geographic coverage
+    * Geographic coverage of the data contained in the dataset. This metadata is directly linked to :ref:`the geographic filter displayed in the front office of the portal <using-the-minimap>` : if the geographic coverage metadata is set for a dataset, users will be able to find it through a geographic search of the catalog.
     * Click the button of the chosen option amongst the following:
 
-      * None *(default)*: the geographic area won't be set for the dataset, which won't be searchable through the mini map.
-      * Auto: the geographic area will be defined as the smallest envelope containing all of the records' locations. Note that all data will be taken into account for this computation regardless of security parameters. The option is only available if the dataset contains geopoints.
-      * Manual: a map appears under this option when it is chosen. Draw a shape on the right geographic area corresponding to the data of the dataset, or paste a GeoJSON after having clicked on the map.
+      * None *(default for datasets without geographical data)*: the geographic coverage won't be set for the dataset, which won't be searchable through the geographic filter.
+      * Auto *(default)*: the geographic coverage will be defined as the smallest envelope containing all of the records' locations. Note that all data will be taken into account for this computation regardless of security parameters. The option is only available if the dataset contains geopoints.
```

Comment:
"Auto" is named "Automatic" now.

Also, the behavior changed, it's available for all datasets for and can fallback to the domain's default location. I guess it can be described with something like this (or something more elegant):

> if the dataset contains geographic data, the geographic coverage will be defined as the smallest envelope containing all of the records' locations. Otherwise, the domain's dataset default location will be used.